visualization/jest-puppeteer.config.js

```diff
@@ -2,7 +2,8 @@ module.exports = {
 	launch: {
 		headless: true,
 		args: ["--allow-file-access-from-files", "--start-maximized"],
-		defaultViewport: { width: 1920, height: 1080 }
-		// slowMo: 250
+		defaultViewport: { width: 1920, height: 1080 }/*,
+		slowMo: process.env.SLOWMO ? process.env.SLOWMO : 0,
```

Comment:
Is the out-commented code still needed?

Reply:
Can be removed, I only left it to be ready when debugging future tests.